styles.css

```diff
@@ -1,38 +1,72 @@
 body {
   margin: 0;
-  border: dashed 1px red;
 }
+
 .wrapper {
   margin: auto;
   width: 80%;
-  border: dashed 1px red;
 }
+
 .header {
   display: flex;
   justify-content: space-between;
-  border: dashed 1px red;
-}
-.logo, .menu {
-  border: dashed 1px red;
 }
+
 .logo {
+  position: relative;
   padding: 40px 0;
+  padding-left: 44px;
+  font: bold 24px/31px pt_serif, arial, helvetica, sans-serif;
+}
+
+.logo__circle {
+  width: 28px;
+  height: 28px;
+  position: absolute;
+  left: 0;
+  top: 50%;
+  transform: translateY(-50%);
+  border-radius: 50%;
+  opacity: .7;
+}
+
+.logo__circle_color_dark-blue {
+  background: #0056EC;
 }
+
+.logo__circle_color_blue {
+  left: 8px;
+  top: 56%;
+  background: #C8DBFB;
```

Comment:
Codacy found an issue: [Expected "#C8DBFB" to be "#c8dbfb" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/LandingPage/pullRequest?prid=6317355)